ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -150,18 +150,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-byo-iam-role
-- as: aws-ipi-byo-iam-role-fips-f28-destructive
```

Comment:
@yunjiang29 aws-ipi-byo-iam-role is already moved to multi-arch ci in this PR, but because arm does not support FIPS, so did not enable fips for this job in  multi-arch ci, do you think it is okay to remove `aws-ipi-byo-iam-role-fips-f14` in the amd ci?

Reply:
or @lwan-wanglin do you think we can enable FIPS on `aws-ipi-byo-iam-role-amd-f28-destructive` for more coverage?

Reply:
yes, we can, so does that mean we enable fips on `aws-ipi-byo-iam-role-amd-f28-destructive`, then remove `aws-ipi-byo-iam-role` from both amd and arm?

Reply:
I think so, two choice:
1. add `aws-ipi-byo-iam-role-arm-f14` and `aws-ipi-byo-iam-role-fips-amd-f28-destructive` in multi-arch, remove `aws-ipi-byo-iam-role` from both amd and arm
2. keep `aws-ipi-byo-iam-role` in amd, add `aws-ipi-byo-iam-role-arm-f14` and `aws-ipi-byo-iam-role-amd-f28-destructive` in multi-arch, remove `aws-ipi-byo-iam-role` from arm.

Reply:
ok, will fix it with option 1.

Reply:
let us wait for @yunjiang29 comment, to see if he has different opinions.

Reply:
> 1. add `aws-ipi-byo-iam-role-arm-f14` and `aws-ipi-byo-iam-role-fips-amd-f28-destructive` in multi-arch, remove `aws-ipi-byo-iam-role` from both amd and arm

Good choice, as FIPS is already included in other profiles and there is not much relevance between arm and byo-iam and FIPS

Reply:
btw, not only `aws-ipi-byo-iam-role`, the same choice can be applied to other ci jobs, `aws-ipi-byo-iam-role` is an example here.

Reply:
Got it. Option 1 looks good.

@lwan-wanglin 
Can you enable AWS SDK install (you can refer to https://github.com/openshift/release/pull/46609) in the following jobs:
```
- as: aws-ipi-byo-iam-role-amd-*
- as: aws-ipi-byo-iam-role-arm-*
- as: aws-ipi-byo-kms-etcd-encryption-amd-*
- as: aws-ipi-byo-kms-etcd-encryption-arm-*
- as: aws-ipi-byo-route53-compact-amd-*
- as: aws-ipi-byo-route53-compact-arm-*
- as: aws-ipi-disconnected-private-sno-amd-*
- as: aws-ipi-disconnected-private-sno-arm-*
- as: aws-ipi-disconnected-sts-basecap-none-amd-*
- as: aws-ipi-disconnected-sts-basecap-none-arm-*
- as: aws-ipi-disconnected-sts-efs-amd-*
- as: aws-ipi-disconnected-sts-efs-arm-*
- as: aws-ipi-disconnected-sts-ep-amd-*
- as: aws-ipi-disconnected-sts-ep-arm-*
- as: aws-ipi-imdsv2-amd-*
- as: aws-ipi-imdsv2-arm-*
- as: aws-ipi-ingress-lbtype-classic-amd-*
- as: aws-ipi-ingress-lbtype-classic-arm-*
- as: aws-ipi-ingress-lbtype-nlb-amd-*
- as: aws-ipi-ingress-lbtype-nlb-arm-*
- as: aws-ipi-ovn-efs-amd-*
- as: aws-ipi-ovn-efs-arm-*
- as: aws-ipi-proxy-sts-amd-*
- as: aws-ipi-proxy-sts-arm-*
- as: aws-ipi-sdn-amd-*
- as: aws-ipi-sdn-arm-*
- as: aws-ipi-usertags-custom-sg-amd-*
- as: aws-ipi-usertags-custom-sg-arm-*
- as: aws-upi-amd-*
- as: aws-upi-arm-*
```

Reply:
sure, is it only supported from 4.15?